warehouse/internal/repo/upload_test.go

```diff
@@ -762,3 +762,70 @@ func TestUploads_InterruptedDestinations(t *testing.T) {
 
 	require.Equal(t, []string{"1", "2", "3"}, ids)
 }
+
+func TestUploads_PendingTableUploads(t *testing.T) {
+	t.Parallel()
+	db := setupDB(t)
+
+	const (
+		uploadID  = 1
+		namespace = "test_namespace"
+		destID    = "1"
+	)
+
+	_, err := db.Exec(`INSERT INTO wh_uploads (id, destination_id, source_id, in_progress, destination_type, status, namespace, schema, created_at, updated_at)
+		VALUES
+		(1, 1, 1, true, 'RS', 'exporting_data', 'test_namespace', '{}', NOW(), NOW()),
+		(2, 1, 1, true, 'RS', 'aborted', 'test_namespace', '{}', NOW(), NOW())
+	`)
+	require.NoError(t, err)
+
+	_, err = db.Exec(`INSERT INTO wh_table_uploads (id, wh_upload_id, table_name, status, error, created_at, updated_at)
+		VALUES
+		(1, 1, 'test_table_1','exporting_data', '{}', NOW(), NOW()),
+		(2, 1, 'test_table_2','exporting_data_failed', 'error loading data', NOW(), NOW())
+	`)
+	require.NoError(t, err)
```

Comment:
This has be modified with repository in this https://github.com/rudderlabs/rudder-server/pull/3087